string cs_name
int16 battery
string batteryStatus
